### PR TITLE
Remove dependency on System.Runtime.InteropServices.RuntimeInformation

### PR DESCRIPTION
**Pull request by kichalla**
@ryanbrandenburg I see that your commit had more changes than this one. Not sure why they were required, but here after making this change I built EF against this change and it built just fine...so I assume all is good to check this in.

Should fix https://github.com/aspnet/Logging/issues/311

### CODE REVIEW

File: src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs [hunk 1]
```diff
@@ -6,6 +6,7 @@
 using System.Runtime.InteropServices;
```
Comment (ryanbrandenburg) [inline src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs]: Doesn't this line need to go too?

**Comment by davidfowl** [comment]
@kichalla I don't think we should make this change. We need to fix RuntimeInformation (on corefx) if it doesn't work.

**Comment by ryanbrandenburg** [comment]
@kichalla the problem I ran into that made me try all the other fixes was that we were getting a mismatched System.Runtime (20 vs 21). If that doesn't show up for you on mono, coreclr, windows mac or linux then guess you're good to go, though I've no idea what makes yours work and mine not.

**Comment by ryanbrandenburg** [comment]
I do agree with @davidfowl that making inerop show up on linux is the preferable fix, though I've no suggestion on how to do that (last time I just got lucky and @anurse knew the answer)

File: src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs (hunk 1 above)
Comment (kichalla) [inline src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs]: :+1:

**Comment by kichalla** [comment]
@davidfowl I agree that probably the right thing to do is fix it in CoreFx but can we file a bug in CoreFx and check this in as a temporarily...we already seem to be using this in several places: 

https://github.com/aspnet/dnx/blob/dev/src/Microsoft.Dnx.Tooling/Program.cs#L46
https://github.com/aspnet/dnx/blob/dev/src/Microsoft.Dnx.Runtime.Sources/Impl/RuntimeEnvironmentHelper.cs

**Comment by muratg** [comment]
This is a simple workaround until CoreFX can get their side done.  Let's just get it in to unblock our cross plat builds.

:shipit:

**Comment by kichalla** [comment]
I pushed this change and MusicStore now builds runs on Mac

**Comment by kichalla** [comment]
f7c9fc32a801bc0c756931adfdca73ab4657727a

**Pull request closed without merging**